hack/release.sh

```diff
@@ -33,7 +33,9 @@ CRI_CTL_PLATFORMS=(
     windows/amd64
     windows/386
     darwin/amd64
+    darwin/arm64
 )
+
 CRI_TEST_PLATFORMS=(
     linux/amd64
     linux/386
```

Comment:
unrelated - k/k stopped building for `linux/arm`. Do we still need it here?

Reply:
Yep, makes sense. Following-up in https://github.com/kubernetes-sigs/cri-tools/pull/1308